src/main/java/com/avispl/symphony/dal/communicator/lumen/vc/tr60a/LumenVCTR60ACommunicator.java

```diff
@@ -0,0 +1,1985 @@
+				case IMAGE_MODE:
+					performControl(PayloadCategory.CAMERA, commandField, PictureMode.getByName(value).getCode());
+					break;
+				case TWO_DNR:
+				case THREE_DNR:
+					performControl(PayloadCategory.CAMERA, commandField, PictureDNROptions.getByName(value).getCode());
+					break;
+				case BRIGHTNESS:
+					float levelBright = Float.parseFloat(value);
+					byte[] pq = convertToTwoBytePQ(levelBright, 14);
+					performControl(PayloadCategory.CAMERA, commandField, (byte) 0x00, (byte) 0x00, pq[0], pq[1]);
+					handlePictureSlider(commandField, Command.BRIGHTNESS_CURRENT, value, LumenVCTR60AConstants.ZERO, LumenVCTR60AConstants.LABEL_END_HUE_LEVEL, LumenVCTR60AConstants.RANGE_START_HUE_LEVEL, LumenVCTR60AConstants.RANGE_END_HUE_LEVEL, stats, advancedControllableProperties);
+					break;
+				case SATURATION:
+					float levelSaturation = Float.parseFloat(value);
+					byte[] saturation = convertToTwoBytePQ(levelSaturation, 14);
+					performControl(PayloadCategory.CAMERA, commandField, (byte) 0x00, (byte) 0x00, saturation[0], saturation[1]);
+					handlePictureSlider(commandField, Command.SATURATION_CURRENT, value, LumenVCTR60AConstants.ZERO, LumenVCTR60AConstants.LABEL_END_HUE_LEVEL, LumenVCTR60AConstants.RANGE_START_HUE_LEVEL, LumenVCTR60AConstants.RANGE_END_HUE_LEVEL, stats, advancedControllableProperties);
+					break;
+				case HUE:
+					float levelHue = Float.parseFloat(value);
+					byte[] hue = convertToTwoBytePQ(levelHue, 14);
+					performControl(PayloadCategory.CAMERA, commandField, (byte) 0x00, (byte) 0x00, hue[0], hue[1]);
+					handlePictureSlider(commandField, Command.HUE_CURRENT, value, LumenVCTR60AConstants.ZERO, LumenVCTR60AConstants.LABEL_END_HUE_LEVEL, LumenVCTR60AConstants.RANGE_START_HUE_LEVEL, LumenVCTR60AConstants.RANGE_END_HUE_LEVEL, stats, advancedControllableProperties);
+					break;
+				case GAMMA:
+					float levelGamma = Float.parseFloat(value);
+					performControl(PayloadCategory.CAMERA, commandField, (byte) levelGamma);
+					handlePictureSlider(commandField, Command.BRIGHTNESS_CURRENT, value, LumenVCTR60AConstants.ZERO, "4", LumenVCTR60AConstants.RANGE_START_HUE_LEVEL, 4F, stats, advancedControllableProperties);
+					break;
+				case SHARPNESS:
+					float levelSharpness = Float.parseFloat(value);
+					byte[] sharpness = convertToTwoBytePQ(levelSharpness, 11);
+					performControl(PayloadCategory.CAMERA, commandField, (byte) 0x00, (byte) 0x00, sharpness[0], sharpness[1]);
+					handlePictureSlider(commandField, Command.BRIGHTNESS_CURRENT, value, LumenVCTR60AConstants.ZERO, "11", LumenVCTR60AConstants.RANGE_START_HUE_LEVEL, 11F, stats, advancedControllableProperties);
+					break;
+				default: {
+					throw new IllegalStateException("Unexpected value: " + commandField);
+				}
+			}
+
+		} catch (Exception e){
+			logger.error("error during command " + commandField.getName() + " send", e);
+			throw new IllegalStateException("Error while sending command " + commandField.getName());
```

Comment:
We should probably also include the original error reference here 
And logging error on top of this may be too much, since any thrown exception will be logged still